src/expressions/if-expr.md

```diff
@@ -125,31 +103,71 @@ if let E::X(n) | E::Y(n) = v {
 }
 ```
 
-The expression cannot be a [lazy boolean operator expression][_LazyBooleanOperatorExpression_].
-Use of a lazy boolean operator is ambiguous with a planned feature change of the language (the implementation of if-let chains - see [eRFC 2947][_eRFCIfLetChain_]).
-When lazy boolean operator expression is desired, this can be achieved by using parenthesis as below:
+The expression cannot be a [lazy boolean operator expression][_LazyBooleanOperatorExpression_]. Scrutinee expressions also cannot be a [lazy boolean operator expression] due to ambiguity and precedence with [chains of expressions].
```

Comment:
These links seem to be broken.

Also, can you keep it to one sentence per line?

Reply:
Done